pkg/pipeline/WorkflowService.go

```diff
@@ -194,7 +201,18 @@ func (impl *WorkflowServiceImpl) SubmitWorkflow(workflowRequest *WorkflowRequest
 		miniCred := []v12.EnvVar{{Name: "AWS_ACCESS_KEY_ID", Value: impl.ciConfig.BlobStorageS3AccessKey}, {Name: "AWS_SECRET_ACCESS_KEY", Value: impl.ciConfig.BlobStorageS3SecretKey}}
 		containerEnvVariables = append(containerEnvVariables, miniCred...)
 	}
-
+	pvc := appLabels[strings.ToLower(fmt.Sprintf("%s-%s", CI_NODE_PVC_PIPELINE_PREFIX, workflowRequest.PipelineName))]
+	if len(pvc) == 0 {
+		pvc = appLabels[CI_NODE_PVC_ALL_ENV]
+	}
+	if len(pvc) != 0 {
+		workflowRequest.IsPvcMounted = true
+		workflowRequest.IgnoreDockerCachePush = true
+		workflowRequest.IgnoreDockerCachePull = true
+	}
+	if impl.ciConfig.IgnoreDockerCacheForCI && workflowRequest.CacheInvalidate {
+		workflowRequest.IsPvcMounted = false
```

Comment:
this is wrong, whats the intent of it.

Reply:
was just doing some checks, didn't need it so removed it